health/health_json.c

```diff
@@ -113,6 +113,28 @@ void health_alarm_log2json(RRDHOST *host, BUFFER *wb, uint32_t after) {
     netdata_rwlock_unlock(&host->health_log.alarm_log_rwlock);
 }
 
+static inline void health_rrdcalc_values2json_nolock(RRDHOST *host, BUFFER *wb, RRDCALC *rc) {
+    (void)host;
+    char value_string[100 + 1];
+    format_value_and_unit(value_string, 100, rc->value, rc->units, -1);
```

Comment:
This is not used since you don't generate `value_string`

Reply:
I forgot to remove from the original function, thank you! :)